diff --git a/android/app/build.gradle b/android/app/build.gradle
index 7f4d5a2..76f4edc 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -46,7 +46,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "phanan.koel.app"
+        applicationId "com.zapar.app"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
         minSdkVersion flutter.minSdkVersion
